internal/parser/scanner/scanner.go

```diff
@@ -169,9 +170,9 @@ func (s *scanner) Next() token.Token {
 	// case '$':
 	// 	return scanDollarSign(s)
 	default:
-		fmt.Println("TBI")
+		fmt.Println("DEF")
```

Comment:
What does this mean? Should an error token be returned? (unhandled rune)

Reply:
Im still not sure what the default condition must be because I dont know what tokens fall under this category.

Reply:
If anything, then unexpectedRune